pkg/util/mount/linux.go

```diff
@@ -47,10 +50,44 @@ func (mounter *mounter) Unmount(target string, flags int) error {
 	return syscall.Unmount(target, flags)
 }
 
+// safe_format_and_mount is a utility script on GCE VMs that probes a persistent disk, and if
+// necessary formats it before mounting it.
+// This eliminates the necesisty to format a PD before it is used with a Pod on GCE.
+// TODO: port this script into Go and use it for all Linux platforms
+type gceSafeFormatAndMount struct {
```

Comment:
Who creates and returns an instance of this type?  AFAICT it is defined and never used?